login.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓 계정</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="common.css" />
+    <link rel="stylesheet" href="login.css" />
+  </head>
+  <body>
+    <main class="main">
+      <a href="/" class="main__logo">
+        <img src="img/logo.svg" alt="판다마켓 로고">
+        판다마켓
+      </a>
+      <article class="main__login-wrap">
+        <form class="login__form">
+          <div class="form__item">
+            <label class="form__label" for="login--id">이메일</label>
```

Comment:
```suggestion
            <label class="form__label" for="email">이메일</label>
```

email을 위한 인풋 그룹인데 login--id라고 하는건 비직관적으로 보여요!